components/camel-kudu/src/main/docs/kudu-component.adoc

```diff
@@ -0,0 +1,91 @@
+[[kudu-component]]
+= Apache Kudu Component
+
+*Since Camel 2.25*
```

Comment:
Looks like shoud be 3.0 as we didn't implenet it for 2.25.0.

Reply:
@oscerd do we decide to have this component for 2.25.0 or only 3.0.0?

Reply:
3.0.0 for the moment